README.md

```diff
@@ -34,9 +34,10 @@ Carefully curated list of various organizations (working in the domain of comput
 - [Jet Brains Campus Ambassador](https://www.jetbrains.com/education/programs/)
 - [GeeksforGeeks Campus Ambassador](https://www.geeksforgeeks.org/campus-ambassador-program-by-geeksforgeeks/)
 - [She++ Campus Ambassador](http://www.sheplusplus.com/ambassadors/)
+- [Product Hunt Campus Ambassador](https://ambassador.producthunt.com/)
```

Comment:
I don't want product hunt to be here its got nothing to do with CS of soft dev please remove it and **amend this commit then I will merge**